doc/sphinx/changes.rst

```diff
@@ -8,6 +8,515 @@ Recent changes
+- **Added:**
+  ``Nat.EvenT`` and ``Nat.OddT`` (almost the same as ``Nat.Even`` and ``Nat.Odd`` but with output in ``Type``.
+  Decidability of parity (with output ``Type``) is provided ``EvenT_OddT_dec`` as well as induction principles ``Nat.EvenT_OddT_rect`` and ``Nat.OddT_EvenT_rect`` (with output ``Type``)
+  (`#15427 <https://github.com/coq/coq/pull/15427>`_,
+  by Olivier Laurent).
+- **Added:**
+  Added a proof of ``sin x < x`` for positive ``x`` and ``x < sin x`` for negative ``x``
+  (`#15599 <https://github.com/coq/coq/pull/15599>`_,
+  by stop-cran).
+- **Added:**
+  decidability typeclass instances for Z.le, Z.lt, Z.ge and Z.gt, added lemmas Z.geb_ge and Z.gtb_gt
+  (`#15620 <https://github.com/coq/coq/pull/15620>`_,
+  by Michael Soegtrop).
+- **Added:**
+  lemmas ``Rinv_inv``, ``Rinv_mult``, ``Rinv_opp``, ``Rinv_div``, ``Rdiv_opp_r``,
+  ``Rsqr_div'``, ``Rsqr_inv'``, ``sqrt_inv``, ``Rabs_inv``, ``pow_inv``,
+  ``powerRZ_inv'``, ``powerRZ_neg'``, ``powerRZ_mult``, ``cv_infty_cv_0``,
+  which are variants of existing lemmas, but without any hypothesis
+  (`#15644 <https://github.com/coq/coq/pull/15644>`_,
+  by Guillaume Melquiond).
+- **Added:**
+  a Leibniz equality test for primitive floats
+  (`#15719 <https://github.com/coq/coq/pull/15719>`_,
+  by Pierre Roux, reviewed by Guillaume Melquiond).
+- **Added:**
+  support for primitive floats in Scheme Boolean Equality
+  (`#15719 <https://github.com/coq/coq/pull/15719>`_,
+  by Pierre Roux, reviewed by Hugo Herbelin).
+
+Infrastructure and dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- **Changed:**
+  Load plugins using `findlib <http://projects.camlcity.org/projects/findlib.html>`_.
+  This requires projects built with ``coq_makefile`` to either provide a
+  hand written ``META`` file or use the ``-generate-meta-for-package`` option
+  when applicable. As a consequence :cmd:`Declare ML Module` now uses plugin
+  names according to ``findlib``, e.g. `coq-aac-tactics.plugin`.
+  ``coqdep`` accepts ``-m META`` and use the file to resolve plugin names to
+  actual file names.
+  (`#15220 <https://github.com/coq/coq/pull/15220>`_,
+  fixes `#7698 <https://github.com/coq/coq/issues/7698>`_,
+  by Enrico Tassi).
+- **Changed:**
+  Minimum supported zarith version is now 1.11
+  (`#15483 <https://github.com/coq/coq/pull/15483>`_
+  and `#16005 <https://github.com/coq/coq/pull/16005>`_
+  and `#16030 <https://github.com/coq/coq/pull/16030>`_,
+  closes `#15496 <https://github.com/coq/coq/issues/15496>`_,
+  by Gaëtan Gilbert and Théo Zimmermann and Jason Gross).
+- **Changed:**
+  Bump the minimum OCaml version to 4.09.0.
+  As a consequence the minimum supported ocamlfind version is now
+  1.8.1 (`#15947 <https://github.com/coq/coq/pull/15947>`_
+  and `#16046 <https://github.com/coq/coq/pull/16046>`_,
+  fixes `#14260 <https://github.com/coq/coq/issues/14260>`_
+  and `#16015 <https://github.com/coq/coq/pull/16015>`_,
+  by Pierre-Marie Pédrot and Théo Zimmermann).
+
+Miscellaneous
```

Comment:
As frequently happens, this section is all about extraction, so I suggest renaming it (and moving the items from the Standard library section that are relevant to extraction here).